litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -151,9 +150,15 @@ func CreateChaosWorkflow(input *model.ChaosWorkFlowInput, r store.StateData) (*m
 	}
 
 	workflow_id := utils.RandomString(32)
+
+	updatedWorkflowManifest, err := utils.EmbedWorkflowIDIntoManifest(workflow_id, input.WorkflowManifest)
+	if err != nil {
+		return nil, err
+	}
+
 	newChaosWorkflow := database.ChaosWorkFlowInput{
 		WorkflowID:          workflow_id,
-		WorkflowManifest:    input.WorkflowManifest,
+		WorkflowManifest:    updatedWorkflowManifest,
```

Comment:
Use this pkg `github.com/tidwall/sjson`. You modified it in a single line.
```
	newWorkflowManifest, _ := sjson.Set(input.WorkflowManifest, "metadata.labels.0.workflow_id", workflow_id)
```

Reply:
Have updated it, with a minor change, using `metadata.labels.workflow_id`